data/DMHiggsAnalysis.cfg

```diff
@@ -13,13 +13,12 @@ SampleName:       sample
 InputFileList:    input_data.txt
 BaseConfig:       HGamAnalysisFramework/HGamRel20p7.config
 Include:          HGamAnalysisFramework/MCSamples.config
-
+NTupleName:	  ggH125_gamgam.NTuple.root
```

Comment:
Yes, getting truth MET either accessing through the truthHandler or directly on the containers or HGamVariables. This met was estimated with all non-interacting particles, not just DM particles.